pennylane/templates/subroutines/prepselprep.py

```diff
@@ -215,6 +200,12 @@ def wires(self):
         """All wires involved in the operation."""
         return self.hyperparameters["control"] + self.hyperparameters["target_wires"]
 
+    def queue(self, context: QueuingManager = QueuingManager):
+        """Append the operator to the Operator queue."""
+        context.remove(self.lcu)
+        context.append(self)
```

Comment:
Dequeue the input `lcu`. This is made possible by us storing the input `lcu` in `__init__`.